GITHUB101.md

```diff
@@ -0,0 +1,94 @@
+
+A good rule of thumb is that if you can succintly put all your thoughts into a couple paragraphs and don't expect a lot of back and forth, it's ok. If it's complicated, other ways of communication will be more efficient.
+
+
+# What is Github?
+Github is a good place where developers write and combine their individual `source code` (computer instructions, the magic bits that power all sorts of electronic apps) with each other so they can collaborate in teams when building complex applications. 
+
+That's only one of the tools they use to get stuff done. While other tools are very technical, Github has a couple of things that are meant to be used by people who are not technical at all.
+
+## Is it free?
+Yes.
+
+Everything in the scope of this project is free and you don't need a credit card or any other means of payment. Although there are some paid features in Github, none of them are necessary.
+
+## Do I need an account?
+There are many ways to participate in this project. Some ways need you to log into github, some don't.
+
+- If you want to report a problem and follow up on it, the best way would be to create an `issue` (more below). That requires a github login.
+- If you want to download the app you do NOT need a github account (see the links at the top of this page). 
+- If you still want to participate but don't want to do it through github, you can still join us on slack (a kind of social chat app that software developers love) at [GDG Melbourne's slack](http://bit.ly/join_gdgslack). Go into the `#general` channel, and some friendly face may be able to help you out. 
+
+You can use any email you wish when signing up - it can be your work email or personal one, there's no drawbacks in using either. The only time it should matter is if you lose the password and need to receive a reset password link. 
+
+Note that the slack `#general` channel is not specific to this app, so different people have different priorities and may even not be aware that this app exists. However if you're stuck or lost that's a good place for asking for help.
+
+## What is an "issue"? How do I create it?
+
+Github helps its users communicate each other through `"issues"`. Think of it as a tunneled conversation about a specific narrow topic. 
+
+We create `issues` for different purposes, such as features we want to have, or bugs that were found. It's very easy to create an issue, and good practice says that you'll need to:
+
+- search or browse for existing issues, so we don't report the same thing twice
+
+After that, hit the green button 
+![Create issue](docs/create_new_issue.png)
+ 
+- create a short, easy to remember title for it
+- create the first comment, explaining as clear and concise as possible what the goal is
+
+### What happens next?
+
+- After an `issue` is created, you and other people can add comments to it
```

Comment:
Might be good to mention the process which is in Readme.MD and how issues flow from medic to design to dev?

Reply:
I guess it would be better to have it only on README to avoid dup'd info? I would even consider removing this section and leaving just the link.